packages/turbo-repository/__tests__/find-packages.test.ts

```diff
@@ -26,4 +27,52 @@ describe("findPackages", () => {
     assert.deepEqual(pkg2.dependencies, []);
     assert.deepEqual(pkg2.dependents, ["apps/app"]);
   });
+
+  it("returns the package for a given path", async () => {
+    const workspace = await Workspace.find(MONOREPO_PATH);
+
+    for (const [filePath, result] of [
+      ["apps/app/src/util/useful-file.ts", "app-a"],
```

Comment:
We should have some tests for root files e.g. a root `tsconfig.json`, `pnpm-lock.yaml`, `pnpm-workspace.yaml`